webapp/src/ts/modules/analytics/analytics-target-aggregates.component.html

```diff
@@ -20,7 +22,14 @@
     <p class="loading-status" *ngIf="!error && !aggregates?.length">{{'analytics.target.aggregates.no.targets' | translate}}</p>
   </div>
 
-  <router-outlet></router-outlet>
+  <div class="content-pane right-pane">
+    <div class="item-content"
+      [ngClass]="{ 'col-sm-6': isSidebarFilterOpen, 'col-sm-8': !isSidebarFilterOpen }">
+    <router-outlet></router-outlet>
+    </div>
```

Comment:
One line
identation
```suggestion
    <div class="item-content" [ngClass]="{ 'col-sm-6': isSidebarFilterOpen, 'col-sm-8': !isSidebarFilterOpen }">
      <router-outlet></router-outlet>
    </div>
```